libpod/runtime_migrate.go

```diff
@@ -0,0 +1,43 @@
+package libpod
+
+import (
+	"context"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+func (r *Runtime) migrate() error {
+	runningContainers, err := r.GetRunningContainers()
+	if err != nil {
+		return err
+	}
+
+	allCtrs, err := r.state.AllContainers()
+	if err != nil {
+		return err
+	}
+
+	logrus.Infof("stopping all containers")
+	for _, ctr := range runningContainers {
+		logrus.Infof("stopping %s", ctr.ID())
+		ctr.Stop()
```

Comment:
We should at least log errors here

Reply:
probably here it is safer to return on error.  I'll change it